swingset/pom.xml

```diff
@@ -96,6 +98,21 @@
 			<artifactId>jdbcrowsetimpl</artifactId>
 			<version>${version.jdbcrowsetimpl}</version>
 		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/org.netbeans.api/org-openide-util-lookup -->
+		<dependency>
+			<groupId>org.netbeans.api</groupId>
+			<artifactId>org-openide-util-lookup</artifactId>
+			<version>${version.netbeans}</version>
+		</dependency>
```

Comment:
I see this is for the CentralLookup class. Mentioned for some boolean constants, but it appears it is used to store state data for Event Bus?

Reply:
As discussed elsewhere, this is a way to make stuff accessible without an API. For the event bus, there is an API in SSUtils.